eesp-ikev2.org

```diff
@@ -233,55 +239,66 @@ per packet basis inside any inner flow to avoid packet reordering.
-
-- Possibility for unidirectional SAs. Compared to [[RFC9611]], when a
-  per-resource SA is established, it is bidirectional. However, both
-  directions of the SA MAY not always be in use. Using CREATE_CHILD_SA
-  does not allow unidirectional SAs.
-
-- No extra setup time, i.e., zero round-trip time to set up
-  additional Sub SAs. This would be more efficient than using large
-  IKE window size specified in [[RFC7296]] to manage multiple SAs.
-
-- Sub SAs are more efficient to create, rekey, and delete. Their
-lifecycle management is simpler compared to traditional Child SAs.
-
-- When using hierarchical key derivation, especially when using
-  hardware key derivation, Sub SA keys can be derived on-the-fly
-  per packet. This reduces "Data-plane performance degradation due to
-  the use of a larger number of keys" as noted in
-  [[I-D.ponchon-ipsecme-anti-replay-subspaces]].
-
-To negotiate Sub SA SUB_SA_ID in Session ID Transform. Or in a Flow
-IDs Transform. TBD: expand Sub SA with Flow ID  negotiation
-
-AEAD transforms, such as AES-GCM or those specified in [[RFC8750]],
-require that the IV does not repeat within a single Sub SA. Each Sub
-SA uses a distinct key, ensuring proper cryptographic separation.
-While the IV may repeat across different Sub SAs, this complies with
-the requirement that each key must be paired with a unique IV.
-
-Each Sub SA MUST maintain an independent sequence number space
-when using full 64-bit sequence numbers. For a specific key within
-a Sub SA, sequence numbers MUST BE unique and follow a
-monotonically increasing order to meet cryptographic requirements.
+* Sub SA
+Existing mechanisms for establishing Child SAs, as described in
+[[RFC7296]], yield bidirectional SAs.  When using
+per resource as  specified [[RFC9611]] or DSCP bidirectional
+SAs are not always necessary; think predomently unidirectional TCP
+or QUIC flow. In many use cases, several uni directinal SAs
+utilized, while others are unused which can result in unnecessary
+overhead for management, rekeying, and resource consumption.
+Furthermore, using multiple bidirectional Child SAs for granular
+traffic flows often leads to additional setup delays and complex
+lifetime management. This inefficiency is particularly acute in
+high-throughput or low-latency environments, where rapid setup and
+teardown of SAs is essential to maintain performance.
+
+An EESP Sub SA provides a unidirectional Security Association
+derived from an existing EESP Child SA pair.  It inherits all of
+the Child SA’s properties except for keys, sequence number space,
+and IV space, each of which MUST be unique to each Sub SA.  By
+defining these unidirectional flows, Sub SAs offer a more efficient
+alternative to large numbers of bidirectional Child SAs with the
+same traffic selectors [[RFC7296]], [[RFC9611]].
+
+Each Sub SA is identified by a Sub SA ID, which MUST be carried in
+each EESP packet—either in the Session ID field or in the Flow ID
```

Comment:
This is a bit complicated. Can we choose a single place to Sub SA ID (e.g. Session ID)? If not, then a justification would be needed.